client/pages/OriginalDataPoint/OriginalDataPoint.tsx

```diff
@@ -28,13 +29,15 @@ const OriginalDataPoint = () => {
         cycleName,
       })
     )
-    return () => dispatch(OriginalDataPointActions.reset())
+    dispatch(OriginalDataPointActions.reset())
   }, [])
 
   if (!originalDataPoint) {
     return null
   }
 
+  if (originalDataPoint.countryIso !== countryIso) return null
```

Comment:
We need to discuss this url mismatch in daily on tuesday (reminder)

Reply:
ok